src/Processor/Gotenberg.php

```diff
@@ -134,7 +137,7 @@ public function getPdfFromString(string $html, array $params = [], bool $returnF
         }
 
         foreach (['header', 'footer'] as $item) {
-            if (isset($params[$item . 'Template'])) {
+            if (isset($params[$item . 'Template']) && method_exists($chromium, $item)) {
```

Comment:
This line  is not necessary due the fix on line [110](https://github.com/pimcore/web-to-print-bundle/pull/57/files#diff-3f744c848f3380e543943525d3e3cca3d90299289c928e75fbf6bd1840418a3fR110)
or did i miss something?

https://github.com/gotenberg/gotenberg-php/blob/de7d14d7e7a7d10d04018156937fec907055e3b3/src/Modules/ChromiumPdf.php#L110-L128

Reply:
This is necessary, because some Configs are removed in Gotenberg 8, so it´s a error prevention in case some only update dependencies and let config settings unchanged.

Reply:
Won't hurt but doesn't seem necessary, could you confirm?

Even the intellisense is fine
![image](https://github.com/pimcore/web-to-print-bundle/assets/6014195/8a448d3e-148e-4c9f-aba5-370defc39587)

![image](https://github.com/pimcore/web-to-print-bundle/assets/6014195/6610b1ac-82c8-4287-89c1-e2f9afcd8b04)

and it's not mentioned in 
![image](https://github.com/pimcore/web-to-print-bundle/assets/6014195/39c908ab-1dd5-44aa-a815-bdab4823f9e5)
https://github.com/gotenberg/gotenberg/releases/tag/v8.0.0

So it's would be fine.

```suggestion
            if (isset($params[$item . 'Template'])) {
```